packages/augur-core/source/contracts/Augur.sol

```diff
@@ -93,6 +93,8 @@ contract Augur is IAugur, IAugurCreationDataGetter, CashSender {
 
     int256 private constant DEFAULT_MIN_PRICE = 0;
     int256 private constant DEFAULT_MAX_PRICE = 1 ether;
+    uint256 private constant RECOMMENDED_TRADE_INTERVAL = 10;
+    uint256 private constant DEFAULT_RECOMMENDED_TRADE_INTERVAL = 10 ** 17;
```

Comment:
These two variables are named very similarly, implying that one is just a default for the other (which is odd when both are constants), but they appear to have different types (one is maybe a fixed point number, while the other is just a normal number).  Recommend renaming these or doing something to improve the clarity as to their purpose.

Reply:
Yea I’ll think more on the naming